src/controllers/timeTaskCtrl.js

```diff
@@ -0,0 +1,112 @@
+const TimeTask = require('../models/timeTaskModel');
+
+exports.createTimeTask = (req, res) => {
+  const newTimeTask = new TimeTask(req.body);
+  newTimeTask
+    .save()
+    .then(() => res.status(201).json({ message: 'TimeTask Created! :', newTimeTask }))
```

Comment:
Remove this please :)
```suggestion
    .then(() => res.status(201).json({ message: 'TimeTask Created!', newTimeTask }))
```